src/game_interpreter.cpp

```diff
@@ -786,7 +786,7 @@ bool Game_Interpreter::CommandShowMessage(RPG::EventCommand const& com) { // cod
 	++index;
 
 	// Check for continued lines via ShowMessage_2
-	while (index < static_cast<int>(list.size()) && list[index].code == Cmd::ShowMessage_2) {
+	while (index < static_cast<int>(list.size()) && static_cast<Cmd>(list[index].code) == Cmd::ShowMessage_2) {
```

Comment:
Maybe we can resolve all these added `static_cast`s in another way after merging?

Reply:
https://github.com/EasyRPG/liblcf/issues/370